_automation/grammars.json

```diff
@@ -7,7 +7,8 @@
       "scanner.cc"
     ],
     "reference": "v0.19.0",
-    "revision": "b6667bed364733c8a8f8e5973749f86cfa04ba2a"
+    "revision": "b6667bed364733c8a8f8e5973749f86cfa04ba2a",
+    "updateBasedOn": "commit"
```

Comment:
any grammar that is referencing a tag should have `"updateBasedOn": "tag"`

Reply:
agree. I'll make this change.